src/foam/nanos/notification/sms/NullSMSService.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.notification.sms',
+  name: 'NullSMSService',
+
+  implements: [
+    'foam.nanos.notification.sms.SMSService'
+  ],
+
+  methods: [
+    {
+      name: 'sendSms',
+      javaCode: 'return smsMessage;',
+      code: function() {
+        return;
```

Comment:
This line is redundant. Just add a line like this to indicate you intend to do nothing (NOP = No-Operation):
// NOP